extensions/guacamole-auth-ldap/doc/examples/exampleConfigGroup.ldif

```diff
@@ -9,3 +9,4 @@ guacConfigParameter: port=5900
 guacConfigParameter: password=secret
 member: cn=user1,dc=example,dc=com
 member: cn=user2,dc=example,dc=com
+seeAlso: ou=admins,ou=groups,dc=example,dc=com
```

Comment:
Yes it comes from groupOfNames: http://www.zytrax.com/books/ldap/ape/#groupofnames